<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatusBarWidget</class>
 <widget class="QWidget" name="StatusBarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="iconLabel">
     <property name="text">
      <string>icon</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="messageLabel">
     <property name="text">
      <string>Status...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="spinnerLabel">
     <property name="text">
      <string>spinner</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
